packages/core-sdk/src/types/resources/royalty.ts

```diff
@@ -185,20 +78,41 @@ export type ClaimAllRevenueRequest = {
     autoUnwrapIpTokens?: boolean;
   };
 };
+export type BatchClaimAllRevenueRequest = WithClaimOptions & {
+  /** The ancestor IPs from which the revenue is being claimed. */
+  ancestorIps: (Omit<ClaimAllRevenueRequest, "ancestorIpId" | "wipOptions"> & {
```

Comment:
should this be `"ancestorIpId" | "claimOptions"`? There is no `wipOptions` in ClaimAllRevenueRequest